### DISCUSSION TITLE
Am I doing something wrong?

### DISCUSSION THREAD

**Discussion by Stuffita** [Kas-tle/java2bedrock.sh#3083] [Converter Help] [answered]
**The model looks dark when it's over a solid block**

![image](https://user-images.githubusercontent.com/58962368/211950626-69ae12a0-3a86-4ad3-8bb9-8bfb66a45e02.png)

**But it looks right when it's over glass or in the air**

![image](https://user-images.githubusercontent.com/58962368/211950692-5ef51109-da7c-46f8-b126-0fe3c109efbc.png)

I'm pretty new to this so sorry if it is something obvious

**Comment by Kas-tle** [comment, marked as answer]
This is a consequence of how armor stands work on bedrock. If the top of the armor stand is within a block the armor stand and anything held by it will be rendered as if it has no light. You can either resolve this with emissive materials or ensuring the armor stand sits higher.

**Reply by HotsukokuAito** [reply]
I have da same problem, what emissive materials do u mean?
